### DISCUSSION TITLE
failure to access the hack together repository template .env.user.local details

### DISCUSSION THREAD

**Discussion by AnthonyByansi** [microsoft/hack-together-teams#55] [Q&A] [answered]
Hey folks, I'm new to Teams Development and I need help on how I can access the PlANNER_GROUP_ID, PLANNER_PLAN_ID and the 
PLANNER_BUCKET_ID to be placed in the .env.user.local 
![SR](https://github.com/microsoft/hack-together-teams/assets/101401469/3114740a-e15f-4cf1-ab37-fd5aeb3c3125)

**Comment by aycabas** [comment, marked as answer]
Hi @AnthonyByansi, great question! Exciting to see your are integrating Planner into your teams app.

Here is what I do to get Planner group id, plan id and bucket id:

- Go to your plan in Microsoft Planner, link will contain both group id and plan id, your link should look like this:
`https://tasks.office.com/microsoft.onmicrosoft.com/en-US/Home/Planner/#/plantaskboard?groupId={YOUR-GROUP-ID}&planId={YOUR-PLAN-ID}`

- Once I get your group id and plan id from the Planner url, I usually go to [Graph Explorer](https://aka.ms/ge), login with my account that has the planner, copy the `plan id` in the following query `https://graph.microsoft.com/v1.0/planner/plans/{plan-id}/buckets`, make sure that I consent required permissions under the **Modify Permissions** and select **Run query**. I should be able to see all the bucket ids under my Planner plan in the response:
![image](https://github.com/microsoft/hack-together-teams/assets/36196437/851f925f-8a97-4d4f-868b-4ad277bf400d)

Let me know if you have any questions along the way. Keep me posted 🙂

**Reply by AnthonyByansi** [reply]
@aycabas thanks for the help.

**Comment by AnthonyByansi** [comment]
I'm new to teams development and my application isn't that ready for submission. But thanks a lot @aycabas

**Reply by waldekmastykarz** [reply]
I suggest you submit anyway. You've got nothing to lose, and we appreciate your effort and would love to recognize it. Even if you didn't get to where you wanted to be, it could still be valuable. You learned something and that's what matters the most. Show us what you've done and grab that badge 🙂

**Reply by AnthonyByansi** [reply]
Ooh thanks @waldekmastykarz, let me do it anyway

**Reply by waldekmastykarz** [reply]
Awesome! Looking forward to seeing it! And don't forget to register in case you haven't done it already, so that we know where we can send your badge 😊
